### DISCUSSION TITLE
Archiving: Buffer to local external HD

### DISCUSSION THREAD

**Discussion by TaylorM87** [ericaltendorf/plotman#682] [Q&A]
I have tried to setup ssh and rsync in a way that will copy finished plots from my nvme drive to a local external hard drive. I can't seem to get ssh or rsync to work let alone with plotman yet. I have setup ssh with passwordless ssh with keyphrase etc. as well as rsync. But ssh username@localhost df -aBK | grep /mnt/plots/ gives df: /run/user/1000/doc: operation not permitted. I have even tried changing sudoers to allow ssh and rsync nopassword. Any help would be so appreciated so I can gain 15 minutes per plot back in time for more plots.

Update1: I have gotten SSH to work correctly by stripping SSH completely out and re-installing and setting up new ssh rsa key pairing. Secondly, I found that I needed to modify the drives permissions so that I can write to the directory and now rsync acts like it's working but isn't actually moving any files.
testfile.test
              8 100%    0.00kB/s    0:00:00 (xfr#1, to-chk=0/1)
Update2: Somehow the authorized_keys file was missing. Created them and input the keys and started working.

**Comment by cherry933** [comment]
A cron job is better to move plots from SSD to local HDD.
